backend/src/main/java/quarano/department/web/QuestionnaireDto.java

```diff
@@ -117,6 +118,11 @@ QuestionnaireDto validate(Errors errors) {
 			errors.rejectValue("dayOfFirstSymptoms", "NotNull.IntialReportDto.dayOfFirstSymptoms");
 		}
 
+		if (dayOfFirstSymptoms != null && guessedDateOfInfection != null
+				&& (guessedDateOfInfection.isEqual(dayOfFirstSymptoms) || guessedDateOfInfection.isAfter(dayOfFirstSymptoms))) {
```

Comment:
Wouldn't `!guessedDateOfInfection.isBefore(dayOfFirstSymptoms)`  be the simpler and more accurate term?

Reply:
Yes :)